spec/spec_helper.rb

```diff
@@ -16,7 +16,7 @@
 Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].each { |f| require f }
 
 # Requires factories defined in lib/solidus_signifyd/factories.rb
-require 'solidus_signifyd/factories'
```

Comment:
did we also move the factories into `lib/extension_name/testing_support/factories`? If yes, we need to do that. In both cases, we need to update the comment here to match the new path.

Reply:
I did! I missed it too scanning thinking I forgot it, but it is in there:

lib/solidus_signifyd/testing_support/factories/factories.rb

Reply:
Doesn't seem fixed.

Reply:
You are correct